src/server.rs

```diff
@@ -171,37 +178,49 @@ struct Opts {
     /// location to store backup files
     #[clap(short, long)]
     dir: Option<String>,
-    /// alternative port location
+    /// alternative uds location
     #[clap(short, long)]
-    port: Option<std::net::SocketAddr>,
+    uds: Option<String>,
 }
 
 #[tokio::main]
 #[allow(unused)]
 pub async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let default_grpc_port = "[::1]:10000".parse().unwrap();
+    let default_uds_path = "/var/tmp/nv-dhcp".to_string();
```

Comment:
please add a note that final desitionation is tbd.  Also, define this in a common place like configf.rs or lib.rs